github-pull-requests.go

```diff
@@ -386,34 +79,27 @@ func populatePRs(url string) {
 	htmlData, err := ioutil.ReadAll(resp.Body)
 	exitOnErr(err)
 
+	const layout = "Jan 2, 2006 at 3:04pm (PST)"
```

Comment:
Then lets move it out and keep as a global constant as `const buggyTimeFormat` or `const buggyTimeLayout`